src/Nova/DomesticAddress.php

```diff
@@ -20,29 +22,35 @@ class DomesticAddress extends BaseResource
         'id',
         'address_line_1',
     ];
-    
+
     public static $title = 'address_line_1';
 
     public function subtitle()
     {
         return "{$this->city->title}, {$this->city->state->abbreviation} {$this->zip_code}";
     }
-    
+
     public static $group = 'Resources';
 
     public function fieldsForIndex(NovaRequest $request)
     {
         return array_filter([
             ID::make()->sortable(),
-            Text::make('Address Line 1', function () {
+            GoogleAutocomplete::make('Address')
+                ->countries('US')
+                ->withValues(['street_number.long_name','locality.long_name','postal_code.short_name','administrative_area_level_1.short_name']),
+            AddressMetadata::make(['Address Line 1'])->fromValue('street_number'),
+            AddressMetadata::make(['City'])->fromValue('locality')->sortable(),
+            AddressMetadata::make(['Zip'])->fromValue('postal_code')->sortable(),
+/*            Text::make('Address Line 1', function () {
                 return $this->address_line_1;
             }),
             Text::make('City', 'city.id', function () {
                 return $this->city->title;
             })->sortable(),
             Text::make('Zip', 'zip.code', function () {
                 return $this->zip->code;
-            })->sortable(),
+            })->sortable(),*/
         ]);
     }
```

Comment:
This shouldn't go into the index fields. Just the regular fields in the nova resource when creating and updating the resource.

Reply:
Note that tests are failing probably because GoogleAutoComplete is expecting the api key in the .env file

Reply:
Yes, I can include it in the GitHub organization secrets